src/backend/appMocks.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * TODO: Consider placing this to src/backend/__mocks__/app.ts
+ * @see https://jestjs.io/docs/en/manual-mocks.html#mocking-user-modules
+ */
+
+import { App } from './app';
+import { AbuseDetectionDBClient } from './abuseDetection';
+
+function notImplemented<T extends object>(target: any = {}): T {
+  const proxy = new Proxy(target, {
```

Comment:
Oh, I'm in love with `Proxy`. So many cool things you can do with them! Gotta be careful though to not put them EVERYWHERE! Here I thought it's a rational thing to use them, since we're on Node 12 and it comes fully supported. Wouldn't necessarily use them on the browser-side just yet if there is need to support older browers. Though `immer` seems to work just fine polyfilling `Proxy`for older browsers (IIRC/AFAIU).